src/main/java/spoon/reflect/visitor/ElementPrinterHelper.java

```diff
@@ -348,9 +348,20 @@ public void writeImports(Collection<CtImport> imports) {
 		}
 	}
 
+	/**
+	 * Write a package statement and a newline.
+	 */
 	public void writePackageLine(String packageQualifiedName) {
+		writePackageStatement(packageQualifiedName);
+		printer.writeln();
+	}
+
+	/**
+	 * Write a package statement.
+	 */
+	public void writePackageStatement(String packageQualifiedName) {
 		printer.writeKeyword("package").writeSpace();
-		writeQualifiedName(packageQualifiedName).writeSeparator(";").writeln();
+		writeQualifiedName(packageQualifiedName).writeSeparator(";");
```

Comment:
Here, I've added the method `writePackageStatement`, as I did not want to break the previous API by modifying `writePackageLine`.